ongebo/java-coders-nairobi
# #155820645 Create layout file for profile details screen

### What does this PR do?
Creates a layout file for the screen to display details for a single user profile
#### Description of the task to be completed?
- Create XML layout file to define a single user's profile
- Add java class for a profile's DetailActivity
- Declare the DetailActivity in AndroidManifest.xml
#### How should this be manually tested?
- Check out this branch and open the project in Android Studio
- Open the `activity_detail.xml` layout resource file in the layout editor
- Compare the implemented layout design to the mockup design
#### Screenshots
![DetailActivityRecording](https://user-images.githubusercontent.com/34346070/54121733-48038f80-440c-11e9-8aaf-26480ca253b1.gif)

![image](https://user-images.githubusercontent.com/34346070/54120151-0bce3000-4408-11e9-8ba7-ec7d6590151b.png)

#### What are the relevant Pivotal Tracker stories?
[#155820645](https://www.pivotaltracker.com/story/show/155820645)
